pos-print/src/main/java/com/clouway/pos/print/adapter/http/HttpBackend.java

```diff
@@ -31,6 +33,17 @@ public void start() {
     ServletContextHandler context = new ServletContextHandler(ServletContextHandler.SESSIONS);
     context.setContextPath("/");
 
+    MongoClientOptions opt = MongoClientOptions
+      .builder()
+      .serverSelectionTimeout(5000)
+      .connectTimeout(300)
+      .socketTimeout(1000)
+      .maxWaitTime(500)
+      .connectionsPerHost(3)
+      .build();
+
+    MongoClient client = new MongoClient("localhost:27017", opt);
```

Comment:
Yes, something like that. 

Kotlin has something similar:
https://github.com/leprosus/kotlin-cli